Floofbot/Services/EventLoggerService.cs

```diff
@@ -73,16 +74,24 @@ private async Task HandleBadMessage(SocketUser user, SocketMessage msg)
         }
         public Task OnMessage(SocketMessage msg)
         {
+            var userMsg = msg as SocketUserMessage;
```

Comment:
I think we'll have to do a null check later on if this returns null.
Can we use an explicit cast `(SocketUserMessage)msg` instead, if this cast will always succeed?

Reply:
I'll test for the `null`. The original function in `CommandHandler.cs` tests it as well.